nano/node/nodeconfig.hpp

```diff
@@ -64,6 +64,11 @@ class node_config
 	std::chrono::milliseconds vote_generator_delay{ std::chrono::milliseconds (100) };
 	unsigned vote_generator_threshold{ 3 };
 	nano::amount online_weight_minimum{ 60000 * nano::Gxrb_ratio };
+	/*
+	 * The minimum vote weight that a representative must have for its vote to be counted.
+	 * All representatives above this weight will be kept in memory!
+	 */
+	nano::amount representative_vote_weight_minimum{ 10 * nano::Gxrb_ratio };
```

Comment:
I picked 10 nanos as mininum vote weight. Is this a good value?

Reply:
`10 * nano::Gxrb_ratio` is 10000 Nano. Was that intentional ?

Here is the [definition](https://github.com/nanocurrency/nano-node/blob/ef985e3cf1dcd026ba8a6df1e6f3f685934e1be9/nano/lib/numbers.hpp#L15) of that value

I tried setting it to 10 Nano instead which results in about 1000 reps on the live network.

Reply:
Good catch, that was a mistake. I've just pushed a fix.